.phpstorm.meta.php/magento_helpers.meta.php

```diff
@@ -24,10 +24,10 @@
             'adminhtml/media_js' => \Mage_Adminhtml_Helper_Media_Js::class,
             'adminhtml/rss' => \Mage_Adminhtml_Helper_Rss::class,
             'adminhtml/sales' => \Mage_Adminhtml_Helper_Sales::class,
-            'api2/data' => \Mage_Api2_Helper_Data::class,
             'api2' => \Mage_Api2_Helper_Data::class,
-            'api/data' => \Mage_Api_Helper_Data::class,
+            'api2/data' => \Mage_Api2_Helper_Data::class,
```

Comment:
I am not sure how is this related to the PR.

Reply:
I think @Judx ran n98-magerun to generate the phpstorm helper because of the new classed that are added including the new module thru composer. and I seem to recall that the order is messed up depending on the php version that's been used, but that same line is just a few lines above so it seems correct.